common/dao-api/src/main/java/org/thingsboard/server/dao/resource/ResourceService.java

```diff
@@ -39,9 +40,9 @@ public interface ResourceService extends EntityDaoService {
 
     ListenableFuture<TbResourceInfo> findResourceInfoByIdAsync(TenantId tenantId, TbResourceId resourceId);
 
-    PageData<TbResourceInfo> findAllTenantResourcesByTenantId(TenantId tenantId, PageLink pageLink);
+    PageData<TbResourceInfo> findAllTenantResourcesByTenantId(TbResourceInfoFilter tbResourceInfoFilter, PageLink pageLink);
```

Comment:
it is better to use simple variable name that is more readable (just ```filter``` instead of ```tbResourceInfoFilter```)